specs/driver_spec.rb

```diff
@@ -72,9 +72,102 @@
       average.must_be :<=, 5.0
     end
 
-    it "returns zero if no trips" do
+    it "returns nil if no trips" do
       driver = RideShare::Driver.new(id: 54, name: "Rogers Bartell IV", vin: "1C9EVBRM0YBC564DZ")
-      driver.average_rating.must_equal 0
+      driver.average_rating.must_equal nil
+    end
+  end
+
+  describe 'calculate_total_rev method' do
+    it 'can return total money driver made' do
+      start_time = Time.parse('2015-05-20T12:14:00+00:00')
```

Comment:
Two interesting cases that apply to all of these describe blocks:

- What if the driver has driven 0 trips?
- What if the driver has an incomplete trip?